multi_drone_sar/Task[] task
---
bool success # indicate successful run of triggered service
string message # informational, e.g. for error messages